.github/workflows/ci.yml

```
@@ -325,15 +352,18 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: '3'
-        if: steps.metadata.outputs.module != 'wasm'
+        if: steps.metadata.outputs.module != 'wasm' &&
+            steps.metadata.outputs.module != 'wasm-wasi-component'
```

You could also do

```suggestion
        if: startsWith(steps.metadata.outputs.module, 'wasm')
```

`startsWith` is a [builtin](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith)